core/modules/file/file.pages.inc

```diff
@@ -1023,7 +1023,16 @@ function file_usage_page($file) {
 
         // Link the label to the URI when possible.
         if (!empty($entity_uri['path'])) {
-          $entity_label = l($label, $entity_uri['path']);
+          // Unlike Node or Term entities File entities have a stream wrapper
+          // to the file set as their path, but the path to the entities' page
+          // is "path/[fid]".
+          if ($entity_type == 'file') {
+            $uri = "file/{$entity_id}";
```

Comment:
Excuse my ignorance, but why the curly syntax here?

Reply:
In fact, it's very unlikely here that php mixes up the string with the variable, so the curly braces probably aren't necessary.
But they don't do any harm, either.

Do you insist in removing them?

Reply:
The typical code style in Backdrop for something like this is concatenation, not interpolation `$uri = 'file/' . $entity_id;`

Reply:
> The typical code style in Backdrop for something like this...

Not sure, but why not.